diff --git a/dist/index.d.ts b/dist/index.d.ts
index 7965bcdf688dc3bf92611684014831ed0535aa9d..8cb176ecde26a939866b2a792761ef5d2ac81aff 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,8 +1,11 @@
 export { defineBddConfig, defineBddProject } from './config';
-export { BDDInputConfig } from './config/types';
+export { type BDDInputConfig } from './config/types';
 export { createBdd } from './steps/createBdd';
-export { test } from './runtime/bddTestFixtures';
+export { test, type BddTestFixtures } from './runtime/bddTestFixtures';
+export { type BddWorkerFixtures } from './runtime/bddWorkerFixtures';
 export { cucumberReporter } from './reporter/cucumber/wrapper';
-export { defineParameterType, IParameterTypeDefinition } from './steps/parameterTypes';
+export { defineParameterType, type IParameterTypeDefinition } from './steps/parameterTypes';
 export { DataTable } from './cucumber/DataTable';
+export type { CucumberStyleStepCtor, CucumberStyleStepFn } from './steps/styles/cucumberStyle';
+export type { PlaywrightStyleStepCtor, PlaywrightStyleStepFn } from './steps/styles/playwrightStyle';
 //# sourceMappingURL=index.d.ts.map
